scripts/build/webpack.config.js

```diff
@@ -64,19 +57,20 @@ const getRules = (env = 'prod') => ({
 
 const baseConfig = {
     context: SOURCE_PATH,
-
     entry: {
         main: './index.ts',
     },
-
+    optimization: {
+        moduleIds: 'named',
+        minimize: true,
```

Comment:
Shall we keep the base (and thus dev) configs un-minimised, and move it to `webpack.prod.conf.js`?

Reply:
Yes, makes sense indeed

Reply:
Ok I just found out that Storybook doesn't run on Webpack 5 so this is back to square one I'm afraid :(